src/core/contract_vm.ml

```diff
@@ -72,7 +69,7 @@ module Lambda = struct
     type t = Raw_repr.Value.t [@@deriving yojson]
   end
   module Interpreter = struct
-    let error_to_string : Lambda_vm.Interpreter.error -> string = function
+    let pp_error : Lambda_vm.Interpreter.error -> string = function
```

Comment:
`pp_` functions usually expect a `Format.formatter` value and return `unit`. I think `<something>_to_string` (or `of_string`) is better in this context.